pytorch_lightning/strategies/deepspeed.py

```diff
@@ -767,13 +767,13 @@ def load_checkpoint(self, checkpoint_path: _PATH) -> Dict[str, Any]:
         return client_state
 
     @property
-    def lightning_restore_optimizer_and_schedulers(self) -> bool:
+    def lightning_restore_optimizer(self) -> bool:
```

Comment:
As with every pr, the reviewer can fulfil their duty by asking "should this be deprecated?" You can choose: 
- no deprecation, remove now 👎 
- remove in v1.7 👍 
- remove in v1.8 🎉 
- remove in v1.9 😄 
- remove in v2.0 👀